openfisca_country_template/variables/benefits.py

```diff
@@ -75,30 +75,32 @@ class parenting_allowance(Variable):
     value_type = float
     entity = Household
     definition_period = MONTH
-    label = ''' Allowance for low income people with children to care for.
-                Loosely based on the Australian parenting pension'''
+    label = "Allowance for low income people with children to care for."
+    documentation = '''
+    Loosely based on the Australian parenting pension.
+    '''
     reference = \
         [u"https://www.servicesaustralia.gov.au/individuals/services/centrelink/parenting-payment/who-can-get-it"]
 
     def formula(household, period, parameters):
         '''
         A person's parenting allowance depends on how many dependents they have,
-        How much they, and their partner, earn
+        how much they, and their partner, earn
         if they are single with a child under 8
         or if they are partnered with a child under 6.
         '''
-        family_income = household('household_income', period)
+        household_income = household('household_income', period)
         income_threshold = 500
-        if family_income > income_threshold:
-            return 0
-        is_single = household.nb_persons(Household.PARENT)
+        income_condition = household_income < income_threshold
```

Comment:
> The operator here was `>`. I updated it to `<` as it seemed strange to check for low income family but to give the allowance only if it was **above** an income threshold 🤔

Reply:
For the order, I also usually do _real_ vs _expected_, in that order.

For the operator, we're changing behaviour, as it should be _<=_ if we stick to the previous implementation.

Here I understand _no more than_ as either _<=_ or _>_ @sarafalamaki.

<img width="1056" alt="Capture d’écran 2021-03-07 à 21 51 52" src="https://user-images.githubusercontent.com/329236/110254418-d5333d80-7f6d-11eb-9b84-3c5aabe733ce.png">

Reply:
I modified the test to map the _no more than_ behaviour, and it fails as expected.

Reply:
@sandcha what do you think ?